silent/tests/testdata/vu-glob.txt

```diff
@@ -0,0 +1,78 @@
+dependabot update -f input.yml --local . --updater-image ghcr.io/dependabot/dependabot-updater-silent
+stdout -count=4 create_pull_request
+pr-created expected-0.json
+pr-created frontend/expected-1.json
+pr-created frontend/expected-2.json
+pr-created backend/expected-3.json
+
+# Testing glob configuration without a group.
+
+-- manifest.json --
+{
+  "dependency-a": { "version": "1.2.3" }
+}
+
+-- expected-0.json --
+{
+  "dependency-a": { "version": "1.2.5" }
+}
+
+-- frontend/manifest.json --
+{
+  "dependency-a": { "version": "1.2.3" },
+  "dependency-b": { "version": "1.2.3" }
+}
+
+-- frontend/expected-1.json --
+{
+  "dependency-a": { "version": "1.2.5" },
+  "dependency-b": { "version": "1.2.3" }
+}
+
+-- frontend/expected-2.json --
+{
+  "dependency-a": { "version": "1.2.3" },
+  "dependency-b": { "version": "1.2.5" }
+}
+
+-- backend/manifest.json --
+{
+  "dependency-a": { "version": "1.2.3" }
+}
+
+-- backend/expected-3.json --
+{
+  "dependency-a": { "version": "1.2.5" }
+}
+
+-- dependency-a --
+{
+  "versions": [
+    "1.2.3",
+    "1.2.4",
+    "1.2.5"
+  ]
+}
+
+-- dependency-b --
+{
+  "versions": [
+    "1.2.3",
+    "1.2.4",
+    "1.2.5"
+  ]
+}
+
+-- input.yml --
+job:
+  package-manager: "silent"
+  source:
+    directories:
+      - "**/*"
+      - "/"
```

Comment:
Should this be only the globstar entry?

Reply:
Normally`**/*` won't match dotfiles or the current working directory `.` which is why I added it here.

We could set `File::FNM_DOTMATCH` which would match `.` but also every directory that starts with `.` will be matched as well. 

I can't think of a reason that would be a bad idea other than it's going to search more directories. It's probably better we set it since it's easy to have glob not match files and directories starting with `.`: `**/[!.]*`

So I'll make that change.